pkgs/build-support/rust/sysroot/src.nix

```
@@ -19,7 +19,7 @@ stdenv.mkDerivation {
   + ''
     ${buildPackages.python3.withPackages (ps: with ps; [ toml ])}/bin/python3 ${./cargo.py}
     mkdir -p $out/src
-    touch $out/src/lib.rs
+    echo '#! [no_std]' > $out/src/lib.rs
```

```suggestion
    echo '#![no_std]' > $out/src/lib.rs
```
would be more idiomatic, right?